src/scripts/provider-check.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+echo "export SLS_ORB_PROVIDER_PARAM=$SLS_ORB_PROVIDER_PARAM" >> "$BASH_ENV"
+###
+# Helper functions
+###
+output_Provider_Selected () {
+  echo "................................."
+  echo " $1 Selected."
+  echo "................................."
+  echo
+}
+
+###
+# Provider check functions
+###
+AWS_Check () {
+  output_Provider_Selected "AWS"
+  if [[ $(command -v aws --version >/dev/null 2>&1) == 1 ]]; then
+    echo "ERROR: AWS CLI NOT INSTALLED"
+    TipMsg_Install_AWS
+    exit 1
+  fi
+  if [[ -z $AWS_ACCESS_KEY_ID || -z $AWS_SECRET_ACCESS_KEY ]]; then
+    if [[ ! -f ~/.aws/credentials ]]; then
+      echo "ERROR: CREDENTIALS NOT FOUND"
+      TipMsg_Install_AWS
+      exit 1
+    fi
+  fi
+  TipMsg_Install_AWS () {
+    echo "
+    ---
+    Tip: Install the AWS CLI and authenticate prior to running the Serverless setup command.
+    Consider using the AWS CLI orb: https://circleci.com/orbs/registry/orb/circleci/aws-cli#usage-install_aws_cli
+
+    Import the AWS CLI Orb To Your Config:
+    ---
+      orbs:
+        aws-cli: circleci/aws-cli@x.y
+    ---
+
+    Run The AWS Setup Command Prior To The Serverless Command:
+    ---
+      steps:
+        - aws-cli/setup
+        - serverless/setup
+    ---
+    "
+  }
+}
+azure_Check () {
+  output_Provider_Selected "Azure"
+  if [[ -z $AZURE_SUBSCRIPTION_ID || -z $AZURE_TENANT_ID || -z $AZURE_CLIENT_ID || -z $AZURE_CLIENT_SECRET ]]; then
```

Comment:
Quote `$AZURE_SUBSCRIPTION_ID`, `$AZURE_TENANT_ID`, `$AZURE_CLIENT_ID` and `$AZURE_CLIENT_SECRET` unless they are intentionally unquoted.